examples/diagnostics_and_criticism/Bayes_factor.ipynb

```diff
@@ -22,7 +22,7 @@
      "name": "stdout",
```

Comment:
<p>As we already discussed<strong>,</strong> Bayes factor<strong>s</strong></p><p> </p><p>much <strong>more</strong> likely</p><p> </p><p>Let<strong>'s</strong> compute it to see <strong><s>it by</s> for </strong>ourselves.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>